src/push/plugin.ts

```diff
@@ -49,8 +49,10 @@ export type PluginData = {
 export type PluginCallback = (data: PluginData) => void;
 
 export function MultiAssetPlugin(callback: PluginCallback): esbuild.Plugin {
+  // Check that the path isn't in an external package by making sure it's at a standard
+  // local filesystem location
   const isBare = (str: string) => {
-    if (str.startsWith('/') || str.startsWith('./') || str.startsWith('../')) {
```

Comment:
@andrewvc Shouldn't we allow relative paths for windows as well?

Reply:
Additionally, I did a quick test and tilde (`~/`) seems to be a valid address to refer to a local package on linux. I think we are not flagging this one correctly as internal dependency either

Reply:
This does work on windows, since at an earlier point it's converted to a forward slash. I ran a test on my local win box with a logging statement:

```
CHECK IS BARE  C:\Users\AndrewCholakian\projects\pushtest\journeys\advanced-example.journey.ts
CHECK IS BARE  ./advanced-example-helpers
CHECK IS BARE  C:\Users\AndrewCholakian\projects\pushtest\journeys\advanced-example.journey.ts
CHECK IS BARE  ./advanced-example-helpers
CHECK IS BARE  C:\Users\AndrewCholakian\projects\pushtest\journeys\example.journey.ts
```

WRT to `~/` I tried it, and found that I can't even run the suite locally using a tilde path, where I can using an absolute path. e.g.

```typescript
import {
  loadAppStep,
  addTaskStep,
  assertTaskListSizeStep,
  checkForTaskStep,
  destroyTaskStep,
} from '~/projects/pushdemo/journeys/advanced-example-helpers';
```

does not work as an import, however replacing the `~` with `/home/andrewvc` does work.

Either way, I don't think it's a common thing, so I'm not too worried about it

Reply:
A slight correction, it appears that win / UNC style paths are just not supported in import statements.

Using the following:

```ts
import { journey } from '@elastic/synthetics';
import {
  loadAppStep,
  addTaskStep,
  assertTaskListSizeStep,
  checkForTaskStep,
  destroyTaskStep,
} from '.\\advanced-example-helpers';
```

results in the error below when running `npx @elastic/synthetics journeys`

```
Error: Cannot find module '.\advanced-example-helpers'
Require stack:
- /home/andrewvc/projects/pushdemo/journeys/advanced-example.journey.ts
- /home/andrewvc/projects/synthetics/dist/loader.js
- /home/andrewvc/projects/synthetics/dist/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/andrewvc/projects/synthetics/node_modules/@cspotcode/source-map-support/source-map-support.js:679:30)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/andrewvc/projects/pushdemo/journeys/advanced-example.journey.ts:2:1)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module.m._compile (/home/andrewvc/projects/synthetics/node_modules/ts-node/src/index.ts:1455:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Object.require.extensions.<computed> [as .ts] (/home/andrewvc/projects/synthetics/node_modules/ts-node/src/index.ts:1458:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/andrewvc/projects/pushdemo/journeys/advanced-example.journey.ts',
    '/home/andrewvc/projects/synthetics/dist/loader.js',
    '/home/andrewvc/projects/synthetics/dist/cli.js'
  ]
}
```

Reply:
I've pushed a comment to the source here explaining this so we can preserve this knowledge.